components/general/SendMsg.tsx

```diff
@@ -0,0 +1,123 @@
+import styles from "@/styles/ContactPage.module.css";
+import { fontSyne, fontRobotoMono } from "@/pages/_app";
+import SocialMedia from "./SocialMedia/SocialMedia";
+import Button from "../base/Button";
+import { useEffect, useState } from "react";
+
+const SendMsg = () => {
+  const [isNameFieldEmpty, setIsNameFieldEmpty] = useState(true);
+  const [isEmailFieldEmpty, setIsEmailFieldEmpty] = useState(true);
+  const [isMessageFieldEmpty, setIsMessageFieldEmpty] = useState(true);
+  const handleNameFieldChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setIsNameFieldEmpty(event.target.value.length == 0 ? true : false);
+  };
+  useEffect(() => {
+    const label = document.getElementById("username_label");
+    if (label) {
+      if (isNameFieldEmpty == false) {
+        label.style.visibility = "hidden";
+      } else {
+        label.style.visibility = "visible";
+      }
+    }
+  }, [isNameFieldEmpty]);
+  const handleEmailFieldChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setIsEmailFieldEmpty(event.target.value.length == 0 ? true : false);
+  };
+  useEffect(() => {
+    const label = document.getElementById("email_label");
+    if (label) {
```

Comment:
Do not use if/ else in this case. Try any of the following:
1Ternary operator
2Short circuit (Using && , || operators)
3Early returns and less nesting
4Function delegation

Reply:
I will refactor this part by using useRef.